tasks/common.yml

```diff
@@ -7,7 +7,6 @@
     file:
       dest: "{{ sensu_config_path }}/conf.d"
       state: directory
-      recurse: true
```

Comment:
This currently fails on OpenBSD just another one of Ansible quirks we have stumbled upon.  To work around this I can just have two tasks 1 for OpenBSD that doesn't have the recurse attribute and one for all others that does.